docs/getting_started.md

```
@@ -1,80 +1,109 @@
+  - [Analysis settings](#configure-analysis-settings)
+- [Starting the RPC Server](#starting-the-rpc-server)
+- [Using Kai for Analysis and Fixes](#using-kai-for-analysis-and-fixes)
+  - [Running Analysis](#running-analysis)
+  - [Solution Server](#solution-server)
+  - [Agent Mode](#agent-mode)
+- [Guided scenarios](#guided-scenarios)
 
 ## Overview
 
-Before you install Kai, you must:
+#### Before you install Kai, you must:
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)